### PR TITLE
New combat robot self maint sound

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Title.

https://user-images.githubusercontent.com/6610922/221024871-5d65eb24-a522-49f9-a7f8-d35f1f1da4fb.mp4

## Why It's Good For The Game

QV reused welding and deconstruction for combat robot self healing, and it's not his fault. I like to expand the soundscape of TGMC so that we don't overuse one ogg.

## Changelog

:cl:
soundadd: New combat robot self maint sounds
/:cl:

### CODE REVIEW

**Comment by Zangeiti** [comment]
sounds like it would get annoying after awhile imo

**Comment by QualityVan** [comment]
Agree that some variation would be good.

**Comment by FIXTHERADBOXPLEASE** [comment]
Reminds me of the leg-repairing sound from EYE.

**Review by Lumipharon — CHANGES_REQUESTED** [review]
If the sound is the same regardless of damage type, take out the sound to play from those if statements.

File: code/datums/status_effects/debuffs.dm [hunk 1]
```diff
@@ -181,15 +181,15 @@
...
 	else if(owner.getFireLoss())
 		owner.heal_limb_damage(0, healing_per_tick, TRUE, TRUE)
-		sound_to_play = 'sound/items/deconstruct.ogg'
+		sound_to_play = 'sound/effects/robotrepair2.ogg'
 	if(!sound_to_play || last_sound)
 		last_sound = FALSE
 		return
 	last_sound = TRUE
-	playsound(owner, sound_to_play, 50)
+	playsound(owner, sound_to_play, 50, 0, 12)
```
Comment (Lumipharon) [inline code/datums/status_effects/debuffs.dm:192]:
```suggestion
	playsound(owner, sound_to_play, 50, sound_range = 12)
```

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**